scss/config/_colors.scss

```diff
@@ -54,9 +54,9 @@ $brand-colors: (
 /// @colors
 /// @group config-colors
 $neutral-colors: (
-  'light-gray': 'brand-blue' ('tint': 80%, 'desaturate': 80%),
-  'gray': 'brand-blue' ('desaturate': 80%),
-  'black': 'brand-blue' ('shade': 30%, 'desaturate': 80%)
+  'light-gray': 'brand-blue' ('tint': 80%, get-function('desaturate'): 80%),
```

Comment:
Why are tint/shade not captured? I wonder if we want some shorthand for this, as it gets quite bulky… :/

Reply:
Because of the exception in accoutrement-color, they don't have to be: https://github.com/oddbird/accoutrement-color/blob/master/sass/_utils.scss#L79-L81

Reply:
ah, figured it out.

Reply:
@jgerigmeyer suppose we'll want similar in accoutrement-scale?

Reply:
I don't think there are any changes needed in accoutrement-scale itself, but users will need similar changes if they pass in functions, yes.